selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsExplorer.java

```diff
@@ -174,6 +174,22 @@ public void selectCommandByName(String commandName) {
     new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
         .until(ExpectedConditions.visibilityOf(getCommandByName(commandName)));
     new Actions(seleniumWebDriver).doubleClick(getCommandByName(commandName)).build().perform();
+    waitCommandIsSelected(commandName);
+  }
+
+  /**
+   * wait until command will be selected
+   *
+   * @param commandName visible name of command in the command explorer
+   */
+  public void waitCommandIsSelected(String commandName) {
+    String locator =
+        String.format(
+            "//div[@id='command_%s' and contains(concat(' ', normalize-space(@class), ' '), ' selected')]",
```

Comment:
Can we avoid "concat(' ', normalize-space(@class), ' ')" here by rewriting the xpath to the:
`//div[@id='command_%s' and contains(normalize-space(@class), 'selected')`
?

Reply:
On my regret no, in this case, this is the best way

Reply:
The simpler way looks even better IMHO:
![screenshot from 2017-11-20 17-37-19](https://user-images.githubusercontent.com/1197777/33026651-958271a0-ce19-11e7-97d6-d3c6d5c541b1.png)